lib/task-data/tasks/delete-raid.js

```diff
@@ -0,0 +1,32 @@
+// Copyright 2015, EMC, Inc.
+/* jshint multistr:true */
+
+module.exports = {
+    friendlyName: 'Delete RAID via Storcli',
+    injectableName: 'Task.Raid.Delete',
+    implementsTask: 'Task.Base.Linux.Commands',
+    options: {
+        deleteAll: true,
+        controller: 0,
+        raidIds: [], /*[
+            {
+                id: 0
+            },
+            {
+                id: 1
+            }
+        ],*/
+        path: '/opt/MegaRAID/storcli/storcli64',
```

Comment:
Seems like it would be useful for us to add `chdir` support or something to the command object so you don't have to do this.